### PR TITLE
Fix for blank hosts when fqdn doesn't return anything

### PR DESCRIPTION
**Pull request by mew1033**

### CODE REVIEW

File: _returners/splunk_nebula_return.py [hunk 1]
```diff
@@ -94,6 +94,8 @@ def returner(ret):
     jid = ret['jid']
     master = __grains__['master']
     fqdn = __grains__['fqdn']
+    # Sometimes fqdn is blank. If it is, replace it with minion_id
+    fqdn = minion_id if fqdn == "" else fqdn
```
Comment (basepi) [inline _returners/splunk_nebula_return.py]: Can we change `if fqdn == ''` to `if not fqdn`? That will handle empty string, but also None or other falsey values. Same with the other two files.

Comment (mew1033) [inline _returners/splunk_nebula_return.py]: How's that?

**Comment by mew1033** [comment]
I made some changes, how's that?

**Comment by basepi** [comment]
Even cleaner than my suggestion! Thanks.

**Pull request merged**